devices/roku-tv.groovy

```diff
@@ -36,9 +37,10 @@ metadata {
         command "home"
         
         command "reloadApps"
-		
-		attribute "application", "string"
-  }
+
+        attribute "application", "string"
+        attribute "app_icon_html", "string"
```

Comment:
Is this an official solution of adding icons to the devices, or is this a hack the makes the dashboards look okay, but may break in the future?